service-api/module/Application/src/Experian/IIQ/ConfigBuilder.php

```diff
@@ -42,9 +42,28 @@ public function buildSAARequest(CaseData $case): array
             'DD' => $dob->format('d'),
         ];
 
-        $saaConfig['ApplicationData'] = [
-            'SearchConsent' => 'Y',
-        ];
+        if (! isset($case->fraudScore)) {
+            throw new RuntimeException('Fraudscore has not been set');
+        }
+
+        if (
+            $case->fraudScore->decision === 'STOP' ||
+            $case->fraudScore->decision === 'REFER'
+        ) {
+                $saaConfig['ApplicationData'] = [
+                    'SearchConsent' => 'Y',
+                    'Product' => '4 out of 4',
+                ];
+        } elseif (
+            $case->fraudScore->decision === 'CONTINUE' ||
+            $case->fraudScore->decision === 'ACCEPT'
+        ) {
+                $saaConfig['ApplicationData'] = [
+                    'SearchConsent' => 'Y',
+                ];
+        } else {
+            throw new RuntimeException('Fraudscore result is not recognised');
```

Comment:
Would be worth including the value of `$case->fraudScore->decision` here to ease debugging (otherwise we won't know what we don't support)